wp-admin/js/widgets/media-audio-widget.js

```diff
@@ -82,11 +113,12 @@
 			metadata = control.mapModelToMediaFrameProps( control.model.toJSON() );
 
 			// Set up the media frame.
-			mediaFrame = wp.media({
+			mediaFrame = new AudioDetailsMediaFrame({
```

Comment:
Having to extend the entire class to remove the `replace-audio` seems like it should be unnecessary. It seems like all we really should have to do is:

```js
mediaFrame.states.remove( 'replace-video' );
```

But it's not working. What also doesn't work is:

```js
mediaFrame.toolbar.view.states.remove( 'replace-video' )
``` 

Nor does this work:

```js
mediaFrame.states.get( 'replace-video' ).deactivate()
```